README.md

```diff
@@ -40,7 +40,7 @@ Read more about the Ansible Service Broker and Ansible Playbook Bundles in this
 * [Other Documentation](docs/README.md)
 
 ## Published Images
-The ansible-service-broker community publishes images in (dockerhub)[https://hub.docker.com/u/ansibleplaybookbundle].
+The ansible-service-broker community publishes images in [Docker Hub](https://hub.docker.com/u/ansibleplaybookbundle).
```

Comment:
Thank you, it's embarrassing that the link formatting was incorrect. Also +1 to using the proper name of the service.